selftests/deployment/roles/common/tasks/repos.yml

```diff
@@ -19,30 +19,6 @@
     - method == 'official' or method == 'copr'
     - ansible_facts['distribution'] == "Fedora"
 
-# Official repos
-- name: Install Avocado EPEL Official Releases repo
-  yum_repository:
-    name: avocado
-    description: Avocado EPEL Official Releases repo
-    baseurl: https://avocado-project.org/data/repos/epel-$releasever-noarch/
-    gpgcheck: yes
-    gpgkey: https://avocado-project.org/data/repos/crosa_redhat_com.gpg
-  when:
-    - method == 'official'
-    - ansible_facts['distribution_file_variety'] == "RedHat"
-    - ansible_facts['distribution_major_version'] == "7"
-
-- name: Install Avocado Fedora Official Releases repo
-  yum_repository:
-    name: avocado
-    description: Avocado Fedora Official Releases repo
-    baseurl: https://avocado-project.org/data/repos/fedora-$releasever-noarch/
```

Comment:
@clebergnu could you please explain a little bit about the history here?
Why where these repositories hosted here and why is not available anymore?

Reply:
> @clebergnu could you please explain a little bit about the history here?

Sure. These repos made the *latest* `RPM` packages available for the Avocado releases.  That wasn't possible before modularity, because Fedora's policy was did not allow bumping major versions.

> Why where these repositories hosted here and why is not available anymore?

With modularity available in Fedora, we've switched strategy and are now delivering the latest packages on `avocado:latest` module stream.

Does that make sense?